<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectTrajectory</class>
 <widget class="QMainWindow" name="ObjectTrajectory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Object Trajectory</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/Image/Image/logo.PNG</normaloff>:/Image/Image/logo.PNG</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LB_View">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string>View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollBar" name="HSB_Zoom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>600</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="AC_Basic"/>
    <addaction name="separator"/>
    <addaction name="AC_Trajectory"/>
    <addaction name="separator"/>
    <addaction name="AC_Automatic_Clustering"/>
    <addaction name="AC_RBNN"/>
    <addaction name="AC_BLOB"/>
    <addaction name="AC_S_CCL"/>
    <addaction name="separator"/>
    <addaction name="AC_K_NN"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="AC_Start"/>
    <addaction name="AC_Stop"/>
   </widget>
   <addaction name="menuMode"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="AC_BLOB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BLOB</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="AC_K_NN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>K-NN</string>
   </property>
   <property name="toolTip">
    <string>K-NN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="AC_Basic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="AC_Trajectory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Trajectory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="AC_Start">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="AC_Stop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F6</string>
   </property>
  </action>
  <action name="AC_S_CCL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S-CCL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="AC_RBNN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RBNN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="AC_Automatic_Clustering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic Clustering</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
